packages/osd-opensearch/src/cli_commands/snapshot.js

```diff
@@ -49,10 +49,12 @@ exports.help = (defaults = {}) => {
       -E                Additional key=value settings to pass to OpenSearch
       --download-only   Download the snapshot but don't actually start it
       --ssl             Sets up SSL on OpenSearch
+      --P               OpenSearch plugin artifact URL to install it on the cluster. We should use the flag multiple times
```

Comment:
Perhaps this can be more clear and I do believe it should be able to allow for `file:path`. Can we verify this? If so, @vagimeli can we get your opinion here.

Reply:
Yes, `file:path` also works just fine. I've also updated the text here. Let me know if it requires any changes.